src/autopas/LogicHandler.h

```diff
@@ -344,7 +346,7 @@ class LogicHandler {
     }
     Particle haloParticleCopy = haloParticle;
     haloParticleCopy.setOwnershipState(OwnershipState::halo);
-    if (not _neighborListsAreValid.load(std::memory_order_relaxed)) {
+    if (forceToContainer) {
       // If the neighbor lists are not valid, we can add the particle.
```

Comment:
Update comment

Reply:
I simply deleted it. I think the new code is self-explanatory.